builtin/providers/aws/resource_aws_elb_load_balancer_backend_server_policy.go

```diff
@@ -0,0 +1,139 @@
+package aws
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsLoadBalancerBackendServerPolicies() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsLoadBalancerBackendServerPoliciesCreate,
+		Read:   resourceAwsLoadBalancerBackendServerPoliciesRead,
+		Update: resourceAwsLoadBalancerBackendServerPoliciesCreate,
+		Delete: resourceAwsLoadBalancerBackendServerPoliciesDelete,
+
+		Schema: map[string]*schema.Schema{
+			"load_balancer_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"policy_names": &schema.Schema{
+				Type:     schema.TypeSet,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Optional: true,
+				Set:      schema.HashString,
+			},
+
+			"instance_port": &schema.Schema{
+				Type:     schema.TypeInt,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceAwsLoadBalancerBackendServerPoliciesCreate(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	loadBalancerName := d.Get("load_balancer_name")
+
+	policyNames := []*string{}
+	if v, ok := d.GetOk("policy_names"); ok {
+		policyNames = expandStringList(v.(*schema.Set).List())
+	}
+
+	setOpts := &elb.SetLoadBalancerPoliciesForBackendServerInput{
+		LoadBalancerName: aws.String(loadBalancerName.(string)),
+		InstancePort:     aws.Int64(int64(d.Get("instance_port").(int))),
+		PolicyNames:      policyNames,
+	}
+
+	if _, err := elbconn.SetLoadBalancerPoliciesForBackendServer(setOpts); err != nil {
+		return fmt.Errorf("Error setting LoadBalancerPoliciesForBackendServer: %s", err)
+	}
+
+	d.SetId(fmt.Sprintf("%s:%s", *setOpts.LoadBalancerName, strconv.FormatInt(*setOpts.InstancePort, 10)))
+	d.Set("load_balancer_name", setOpts.LoadBalancerName)
```

Comment:
It is typical not to set the values back to state in the Create func. We usually have the Create call the Read func (where we make the actual GET request) and set them back to state there